Description: Disable webdav module (source code build and javadoc package)
 because jackrabbit JCR provider is not yet in Debian.
Author: Damien Raude-Morvan <drazzib@debian.org>
Forwarded: not-needed
--- a/core/pom.xml
+++ b/core/pom.xml
@@ -207,6 +207,36 @@
 
     <plugins>
       <plugin>
+        <artifactId>maven-compiler-plugin</artifactId>
+        <executions>
+          <execution>
+            <id>default-compile</id>
+            <phase>compile</phase>
+            <configuration>
+              <excludes>
+                <exclude>**/webdav/*</exclude>
+              </excludes>
+            </configuration>
+            <goals>
+              <goal>compile</goal>
+            </goals>
+          </execution>
+          <execution>
+            <id>default-testCompile</id>
+            <phase>test-compile</phase>
+            <configuration>
+              <testExcludes>
+                <exclude>**/webdav/test/*</exclude>
+              </testExcludes>
+            </configuration>
+            <goals>
+              <goal>testCompile</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+
+      <plugin>
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-jar-plugin</artifactId>
         <executions>
--- a/pom.xml
+++ b/pom.xml
@@ -247,6 +247,24 @@
           </excludes>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-javadoc-plugin</artifactId>
+        <version>${commons.javadoc.version}</version>
+        <configuration>
+          <tags>
+            <tag>
+              <name>todo</name>
+              <!-- todo tag for all places -->
+              <placement>a</placement>
+              <head>To Do:</head>
+            </tag>
+          </tags>
+          <aggregate>true</aggregate>
+          <excludePackageNames>*.webdav.*</excludePackageNames>
+        </configuration>
+      </plugin>
+
     </plugins>
   </build>
 
